paper.md

```diff
@@ -0,0 +1,129 @@
+---
+title: 'Imbalance: A comprehensive, multi-interface, extensively documented Julia toolbox to address class imbalance'
+tags:
+  - machine learning
+  - classification
+  - class imbalance
+  - resampling
+  - oversampling
+  - undersampling
+  - julia
+
+authors:
+  - name: Essam W. Amin
+    orcid: 0009-0009-1198-7166
+    equal-contrib: true
+    affiliation: 1
+  - name: Anthony Blaom
+    orcid: 0000-0000-0000-0000
+    equal-contrib: true 
+    affiliation: 2
+
+affiliations:
+ - name: Cairo University, Egypt
+   index: 1
+ - name: University of Auckland, New Zealand
+   index: 2
+
+date: 11 October 2023
+bibliography: paper.bib
+---
+
+# Summary
+
+Given a set of observations that each belong to a certain class, supervised classification aims to learn a classification model that can predict the class of a new, unlabeled observation [@Cunningham:2008]. This modeling process finds extensive application in real-life scenarios, including but not limited to medical diagnostics, recommendation systems, credit scoring, and sentiment analysis.
+
+In various real-world scenarios, such as those pertaining to the detection of particular conditions like fraud, faults, pollution, or rare diseases, a severe discrepancy between the number of observations in each class can occur. This is known as class imbalance. This poses a problem if assumptions inherent in the classification model imply hindered performance when the model is trained on imbalanced data as is commonly the case [@Ali:2015]. Two prevalent strategies for mitigating class imbalance, when it poses a problem to the classification model, involve either increasing the representation of data in the smaller classes through oversampling or reducing the data in the larger classes through undersampling. It may be also possible to achieve even greater performance by combining both approaches [@Zeng:2016] or by resampling the data multiple times and training the classification model on each resampled dataset to form an ensemble model that aggregates results from different model instances [@Liu:2009].
```

Comment:
```suggestion
In various real-world scenarios, such as those pertaining to the detection of particular conditions like fraud, faults, pollution, or rare diseases, a severe discrepancy between the number of observations in each class can occur. This is known as class imbalance. This poses a problem if assumptions inherent in the classification model imply hindered performance when the model is trained on imbalanced data as is commonly the case [@Ali:2015]. Two prevalent strategies for mitigating class imbalance, when it poses a problem to the classification model, involve either increasing the representation of less frequently occurring classes through oversampling or reducing the data in the larger classes through undersampling. It may be also possible to achieve even greater performance by combining both approaches [@Zeng:2016] or by resampling the data multiple times and training the classification model on each resampled dataset to form an ensemble model that aggregates results from different model instances [@Liu:2009].
```